src/common/tga.cpp

```diff
@@ -36,23 +40,28 @@ void WriteTGA(
 	const unsigned char* const palette,
 	const char* const file_name )
 {
+	std::vector<std::array<uint8_t, 4>> pixels(width * height);
+	size_t i = 0;
+	for(auto & p : pixels)
+		p = { data[ i * 4 + 2 ], data[ i * 4 + 1 ], data[ i * 4 + 0 ], data[ i++ * 4 + 3 ] };
```

Comment:
Let's add a separate function for writing true color TGA, instead of changing old one, that writes paletted TGA.

Reply:
Changed the CaptureScreen code not to modify the palette writing code as intermediate solution to implement TGA version 2.0 support as mentioned in #59.